reactjs-todo/webpack.config.js

```diff
@@ -8,16 +8,15 @@ module.exports = () => {
   const localEnv = dotenv.config().parsed || {};
 
   // Use process environment variables for prod, but fallback to local .env for dev
-  const AM_URL = process.env.AM_URL || localEnv.AM_URL;
   const APP_URL = process.env.APP_URL || localEnv.APP_URL;
   const API_URL = process.env.API_URL || localEnv.API_URL;
   const DEBUGGER_OFF = process.env.DEBUGGER_OFF || localEnv.DEBUGGER_OFF;
   const DEVELOPMENT = process.env.DEVELOPMENT || localEnv.DEVELOPMENT;
   const JOURNEY_LOGIN = process.env.JOURNEY_LOGIN || localEnv.JOURNEY_LOGIN;
   const JOURNEY_REGISTER = process.env.JOURNEY_REGISTER || localEnv.JOURNEY_REGISTER;
   const WEB_OAUTH_CLIENT = process.env.WEB_OAUTH_CLIENT || localEnv.WEB_OAUTH_CLIENT;
-  const REALM_PATH = process.env.REALM_PATH || localEnv.REALM_PATH;
-
+  const WELLKNOWN_URL = process.env.WELLKNOWN_URL || localEnv.WELLKNOWN_URL;
```

Comment:
We probably need to remove REALM_PATH in the `main` branch as well, we should create something for this

Reply:
yeah let's keep any main change is out of this scope. I will attend to the `start` branch after we are done on this.

We can keep an eye for MAIN improvements as part of another ticket